.gitignore

```diff
@@ -50,3 +50,6 @@ x-pack/elastic-agent/pkg/agent/operation/tests/scripts/serviceable-1.0-darwin-x8
 # Terraform
 *.terraform
 *.tfstate*
+
+# Bump
```

Comment:
Nit. It may not be clear to what bump it refers.

Reply:
```suggestion
# Files generated with the bump elastic stack version automation
```